src/figuresClasses.ts

```diff
@@ -1,19 +1,108 @@
+export enum Shape {
+  Triangle = 'triangle',
+  Circle = 'circle',
+  Rectangle = 'rectangle',
+}
+
+export type Color = 'red' | 'green' | 'blue';
+
 export interface Figure {
+  shape: Shape;
+  color: Color;
+  getArea(): number;
+}
 
+function roundDown(num: number): number {
+  return Math.floor(num * 100) / 100;
 }
 
 export class Triangle implements Figure {
+  color: Color;
+
+  shape: Shape = 'triangle';
+
+  a: number;
+
+  b: number;
+
+  c: number;
+
+  constructor(
+    public figureColor: Color,
+    public sideA: number,
+    public sideB: number,
+    public sideC: number,
```

Comment:
```suggestion
    public color: Color,
    public a: number,
    public b: number,
    public c: number,
```

Reply:
using word `public` is a typescript feature, it's creating property with the same name and setting received value

Reply:
fix everywhere